sys/hashes/sha256.c

```diff
@@ -254,14 +81,7 @@ void sha256_update(sha256_context_t *ctx, const void *data, size_t len)
  */
 void sha256_final(sha256_context_t *ctx, void *dst)
 {
-    /* Add padding */
-    sha256_pad(ctx);
-
-    /* Write the hash */
-    be32enc_vect(dst, ctx->state, 32);
-
-    /* Clear the context state */
-    memset((void *) ctx, 0, sizeof(*ctx));
+    sha2xx_final(ctx, dst, SHA256_DIGEST_LENGTH);
```

Comment:
Same as the other, does it make sense to provide these as inline in the header file?